esx_service/cli/vmdkops_admin.py

```diff
@@ -527,14 +531,18 @@ def ls_dash_c(columns, tenant_reg):
 
 def all_ls_headers():
     """ Return a list of all header for ls -l """
-    return ['Volume', 'Datastore', 'Created By VM', 'Created',
+    return ['Tenant', 'Volume', 'Datastore', 'Created By VM', 'Created',
```

Comment:
I would vote for displaying Volume Name as the first column ... can you please accommodate the change?